tools/prettyJSON.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python2
```

Comment:
We don't actually need this; one can simply `cat foo.json | python -m 'json.tool'`